packages/vuetify/src/mixins/registrable/index.ts

```diff
@@ -15,10 +16,10 @@ export type Registrable<T extends string, C extends VueConstructor | null = null
 
 export function inject<
   T extends string, C extends VueConstructor | null = null
-> (namespace: T, child?: string, parent?: string): Registrable<T, C> {
+> (namespace: T, child?: string, parent?: string, nestingOptional?: boolean): Registrable<T, C> {
```

Comment:
Child/parent are only used for the warning, if it's optional you just don't use those arguments.

Reply:
Got it. I did this because inputs are often used without a v-form so most users will see a warning that states a component "must" be used inside a v-form so even though it's a warning it reads more like an error which is misleading. Also without this 322 tests fail.

I'm good with removing this and updating all of the tests to mock the provide but I just want to make sure showing this warning to everyone will be desired behavior. Maybe the warning could be reworded to read less like an error?

Reply:
Thanks I missed that. Fixed now 👍